Cairo Desktop/Cairo Desktop/CairoSettingsWindow.xaml

```diff
@@ -172,6 +177,10 @@
                         Name="chkEnableAutoUpdates">
                         <Label Content="{Binding Path=(l10n:DisplayString.sSettings_General_UpdateCheck)}" />
                     </CheckBox>
+                    <CheckBox Click="ChkRunAtLogOn_Click"
+                        x:Name="chkRunAtLogOn" IsChecked="{Binding RunAtLogOn}">
```

Comment:
It is reading its checked state from a stored setting, but this is not sustainable, since it can be mis-matched if another program modifies the registry. It should check the registry and not be a stored setting.